observability/logging/logging-6.1/6x-cluster-logging-collector-6.1.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: cluster-logging-collector-6-0
+[id="cluster-logging-collector-6-0"]
+= Configuring the logging collector
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+{logging-title-uc} collects operations and application logs from your cluster and enriches the data with Kubernetes pod and project metadata.
+All supported modifications to the log collector are performed though the `spec.collection` stanza in the `ClusterLogForwarder` custom resource (CR).
+
+include::modules/log6x-creating-logfilesmetricexporter.adoc[leveloffset=+1]
+
+include::modules/log6x-cluster-logging-collector-limits.adoc[leveloffset=+1]
+
+[id="cluster-logging-collector-input-receivers_{context}"]
+== Configuring input receivers
+
+The {clo} deploys a service for each configured input receiver so that clients can write to the collector. This service exposes the port specified for the input receiver. For log forwarder `ClusterLogForwarder` CR deployments, the service name is in the format `<ClusterLogForwarder_CR_name>-<input_name>`.
```

Comment:
```suggestion
The {clo} deploys a service for each configured input receiver so that clients can write to the collector. This service exposes the port specified for the input receiver. For log forwarder `ClusterLogForwarder` CR deployments, the service name is in the `<clusterlogforwarder_resource_name>-<input_name>` format.
```

Same comments as earlier.

Reply:
done